pep-0001.txt

```diff
@@ -603,11 +603,9 @@ The Type header specifies the type of PEP: Standards Track,
 Informational, or Process.
 
 The format of a PEP is specified with a Content-Type header.
-Valid values are ``text/plain`` for plaintext PEPs (see :pep:`9`)
-and ``text/x-rst`` for reStructuredText PEPs (see :pep:`12`).
-All new and active PEPs must use reStructuredText, but for backwards
-compatibility, plain text is currently still the default if no
-Content-Type header is present.
+All PEPs now use reStructuredText (see :pep:`12`),
+and have a value of ``text/x-rst``, the default.
+Previously, plaintext PEPs used ``text/plain`` (see :pep:`9`).
```

Comment:
This should clearly explain whether `text/plain` will be accepted or not going forward.

Reply:
I'd remove this line, PEP-0009 itself has a notice stating that it is withdrawn.

Reply:
If people think this should simply be removed, I can remove it. Otherwise, I could revise the above to say "All PEPs _must_ use reStructuredText"...

Reply:
Yes, I think the _must_ language is worth adding.

Reply:
Added above, thanks